domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterAlphaKenyaModule.kt

```diff
@@ -169,7 +169,7 @@ class PlatformParameterAlphaKenyaModule {
     platformParameterSingleton: PlatformParameterSingleton
   ): PlatformParameterValue<Boolean> {
     return platformParameterSingleton.getBooleanPlatformParameter(
-      ENABLE_PERFORMANCE_METRICS_COLLECTION
+      PERFORMANCE_METRICS_COLLECTION
```

Comment:
Why rename this while the qualifier and the default value constant remain unchanged? Ditto in corresponding files.